.github/dependabot.yml

```diff
@@ -0,0 +1,14 @@
+version: 2
+updates:
+  # Enable version updates for Gradle
+  - package-ecosystem: "gradle"
+    # Look for `build.gradle` file in the `root` directory
+    directory: "/"
+    # Check for updates every day (weekdays)
+    schedule:
+      interval: "daily"
+    allow:
+      # Allow updates for Exactpro packages
+      - dependency-name: "com.exactpro*"
```

Comment:
question: why only our dependencies? Shouldn't we update all the dependencies if we can?